src/vrapp/VolumeVisualizationApp.cpp

```diff
@@ -116,54 +114,57 @@ void VolumeVisualizationApp::onCursorMove(const VRCursorEvent& event)
       glm::vec2 pos2d(event.getPos()[0], event.getPos()[1]);
       m_vrVolumeApp->mouse_pos_event(pos2d);
     }
-
-
   }
 }
 
-#define count_Packages 
+#define count_Packages
```

Comment:
`COUNT_PACKAGES`?

Reply:
This is for the multiuser/network plugin,

Reply:
let's keep it